packages/cli/src/Init.ts

```diff
@@ -19,6 +19,23 @@ import { isError } from 'util'
 import { printError } from './utils/prompt/utils/print'
 
 export const defaultSchema = (provider: ConnectorType = 'postgresql') => {
+  // add preview flag
+  if (provider === 'cockroachdb') {
+    return `// This is your Prisma schema file,
+    // learn more about it in the docs: https://pris.ly/d/prisma-schema
+    
+    generator client {
+      provider        = "prisma-client-js"
+      previewFeatures = ["${provider}"]
+    }
+    
+    datasource db {
+      provider = "${provider}"
+      url      = env("DATABASE_URL")
+    }
```

Comment:
I believe this will result in extra spacing starting from the second line in the generated schema file.

Reply:
I think we previously had similar for for MongoDB which was _just_ removed a few days ago. Might also be worth to just quickly go back and check how we did this before.

(Also might be worth to extract the list of providers that should get a preview feature flag to a constant, then we can just keep this around next time - although no provider might be in preview at that time)

Reply:
I believe @aqrln is right, the `generator` and `datasource` blocks are shifted compared to [how you folks did for MongoDB](https://github.com/prisma/prisma/pull/12660/files#diff-3a2fd0c2f24931daa62a694bc42215135099cded6cc9da76ef7c64db4fb6f10f). New PR incoming to fix it